.github/workflows/build.yaml

```diff
@@ -45,15 +45,15 @@ jobs:
       date: ${{ inputs.date }}
       sha: ${{ inputs.sha }}
       matrix_filter: map(select(.CUDA_VER | startswith("11") | not))
-  upload-conda:
-    needs: [conda-cpp-build, conda-python-build]
-    secrets: inherit
-    uses: rapidsai/shared-workflows/.github/workflows/conda-upload-packages.yaml@cuda-12.9.0
-    with:
-      build_type: ${{ inputs.build_type || 'branch' }}
-      branch: ${{ inputs.branch }}
-      date: ${{ inputs.date }}
-      sha: ${{ inputs.sha }}
+  # upload-conda:
+  #   needs: [conda-cpp-build, conda-python-build]
+  #   secrets: inherit
+  #   uses: rapidsai/shared-workflows/.github/workflows/conda-upload-packages.yaml@cuda-12.9.0
+  #   with:
+  #     build_type: ${{ inputs.build_type || 'branch' }}
+  #     branch: ${{ inputs.branch }}
+  #     date: ${{ inputs.date }}
+  #     sha: ${{ inputs.sha }}
```

Comment:
I actually do want to upload the cpp package, but I don't know how to do that. Ideas welcome.

Reply:
I think you should be able to add:
```
      skip_upload_pkgs: rapidsmpf
```
(and maybe `librapidsmpf-tests`) instead of deleting the step.  E.g. [used here](https://github.com/rapidsai/cuvs/blob/6bba2ef71560c9a12fbaddcb083f5d80ca90933d/.github/workflows/build.yaml#L98).